package.json

```diff
@@ -39,7 +39,7 @@
     "nodes": "node ./tasks/testnet.js",
     "proxy": "caddy",
     "stargate": "./builds/Gaia/darwin_amd64/gaiacli rest-server --laddr 'tcp://localhost:9070' --home './builds/testnets/local-testnet/cli_home' --node 'http://localhost:26657' --chain-id 'local-testnet' --trust-node true",
-    "frontend": "cross-env NODE_ENV=development webpack-dev-server --hot --colors --config webpack.renderer.config.js --port 9080 --content-base app/dist --https --mode=development",
+    "frontend": "cross-env NODE_ENV=development FAUCET='' webpack-dev-server --hot --colors --config webpack.renderer.config.js --port 9080 --content-base app/dist --https --mode=development",
```

Comment:
would be nice to have the faucet in development as to develop with the button mind. what do you think?

Reply:
Yeah we can keep it, I'll make him ping localhost at some port, so it won't really give money, alternatively if we want a real faucet also on localhost we can setup a tiny telnet script listening on some port and when pinged he behaves like the remote faucet, or just recycle those scripts with some parameters to make them work locally

Reply:
I'll override it to localhost (cheap solution to have it visible but not burning money from the real faucet to addresses on localhost)